### ISSUE TITLE
Resource not found error

### ISSUE THREAD

**Issue by nazarii-semeniuk** [achievements-app/psn-api#164]
Hi, I've started to receive this error today - "Resource not found". Looks like Sony have changed something in their API. Error occurs when I'm trying to call getUserTrophiesEarnedForTitle for Ghost of Tsushima (npcommunicationId is NPWR22859_00).

I decided to dig deeper, and found out that game's trophyTitlePlatform is "PS5,PSPC", so looks like "PSPC" platform was added. Can I receive this error because of this new platform value?

**Comment by wescopeland** [comment]
Hi @nazarii-semeniuk ,

Do you experience this error only with Ghost of Tsushima, or with all titles when using the function?

**Comment by nazarii-semeniuk** [comment]
Only with Ghost of Tsushima, other games that I have in my profile works greate.

**Comment by nazarii-semeniuk** [comment]
I'm so sorry, that I've disturbed you @wescopeland , it was my fault. I had a check in my code if game platform is equals 'PS5'. If no, I'm adding `{ npServiceName: "trophy" } `param to get info for olders games. Obviously, this new "PS5,PSPC" was failing this check, and I was adding `{ npServiceName: "trophy" } `param.

I decided that it was lib's fault because it was working for a year now, and then just suddenly started to failing.

Thank you for what you're doing. Lib is great :D
